src/blueprint/azext_blueprint/_params.py

```diff
@@ -85,6 +86,11 @@ def load_arguments(self, _):
     with self.argument_context('blueprint list') as c:
         pass
 
+    with self.argument_context('blueprint export') as c:
+        c.argument('blueprint_name', options_list=['--name', '-n'], help='Name of the blueprint definition to export.')
+        c.argument('output_path', type=file_type, help='The directory path for json definitions of the blueprint and artifacts. The blueprint definition file will be named blueprint.json. Artifacts json files will be in a subdirectory named artifacts.', completer=FilesCompleter())
+        c.argument('force', arg_type=get_three_state_flag(invert=True, return_label=True), options_list=['--force', '-f'], help='When set, if directory does not exist, it will be created. If the directory exists and has contents, they will be overwritten. If not set, user will be prompted for permission to proceed')

```

Comment:
`-Force` is the common name used for Powershell cmdlet option to skip user confirmation, here I think you can use the Azure CLI equivalent option name `--yes`. 
`get_three_state_flag` is usually used for resource properties that will be persisted in service side while `action='store_true'` is used to control command behavior (see more in [Registering Flags](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags)): 
```suggestion
        c.argument('force', action='store_true', options_list=['--yes', '-y'], help='When set, if directory does not exist, it will be created. If the directory exists and has contents, they will be overwritten. If not set, user will be prompted for permission to proceed')

```